Nickel/Interfaces/Content/IModParts.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using System.IO;
+
+namespace Nickel;
+
+public interface IModParts
+{
+	IPartEntry RegisterPart(string name, Spr onPart, Spr? offPart);
```

Comment:
I'd probably make it part of `IModShips`. Parts make no sense outside ships anyway